kuma/javascript/src/last-modified.jsx

```diff
@@ -28,21 +26,25 @@ export default function LastModified({
     };
 
     return (
-        <section className="contributors-sub">
-            <ClockIcon />
-            <header>
-                <h4>{gettext('Last modified:')}</h4>
+        <section className="document-meta">
+            <header className="visually-hidden">
+                <h4>Metadata</h4>
             </header>{' '}
-            <time dateTime={lastModified}>
-                {lastModifiedDate.toLocaleString(
-                    documentLocale,
-                    dateStringOptions
-                )}
-            </time>
-            ,{' '}
-            <a href={`${wikiRevisionHistoryURL}`} rel="nofollow">
-                {gettext('by MDN contributors')}
-            </a>
+            <ul>
+                <li className="last-modified">
+                    <b>{gettext('Last modified:')}</b>{' '}
```

Comment:
It is unfortunate that we have all of those `{' '}`, would be a terrible idea to include the space inside the `gettext` call, for example:

```suggestion
                    <b>{gettext('Last modified: ')}</b>
```

Reply:
Unfortunately if i add the space it doesn't translate properly.

Reply:
Yeah, the `{' '}` after the `</b>` is better otherwise the translators will have to deal with a whitespace.

Reply:
> would be a terrible idea to include the space inside the `gettext` call, for example:

@schalkneethling ?? Yes, it would be terrible. Was it a mistake to make the suggested edit?

Reply:
It was totally just a suggestion, it can be safely ignored ;)